website/docs/gitops-templates/quickstart-templates.mdx

```diff
@@ -9,7 +9,7 @@ import TierLabel from "../_components/TierLabel";
 # Quickstart GitOps Templates <TierLabel tiers="Enterprise" />
 
 `Quickstart` templates are [`GitOpsTemplate`s](https://docs.gitops.weave.works/docs/gitops-templates/templates/)
```

Comment:
Unrelated to your changes but this should be a relative link, it currently points to a v0.18.0 document. (can be in a follow-up PR)